pkg/parser/node.go

```diff
@@ -298,7 +304,14 @@ func (n *Node) process(p *types.Event, ctx UnixParserCtx) (bool, error) {
 	if n.Name != "" {
 		NodesHitsOk.With(prometheus.Labels{"source": p.Line.Src, "name": n.Name}).Inc()
 	}
-	if len(n.Statics) > 0 {
+	if hasWhitelist && isWhitelisted && len(n.Statics) > 0 {
+		clog.Debugf("+ Processing %d statics", len(n.Statics))
+		// if all else is good in whitelist, process node's statics
+		err := ProcessStatics(n.Statics, p, clog)
+		if err != nil {
+			clog.Fatalf("Failed to process statics : %v", err)
+		}
+	} else if len(n.Statics) > 0 && !hasWhitelist {
```

Comment:
to be picky, can you make it one `if` to avoid copy/pasta the code ?

Reply:
done